python_training/mimic.py

```diff
@@ -0,0 +1,122 @@
+
+class MyList:
+    """
+    Creates a series of objects that mimic a pythonic list.
+    Each object holds a value, and a pointer to the next item of the list.
+    The user accesss the list through the first item.
+    The first initiation of the class (by the user) creates an empty list, and items are added by calling the append
+    method.
+    """
+    def __init__(self, index=0, value=None):
+        """
+        Creates a list member that holds a specific value of the list, and points to the next value.
+        At first self.next_value is empty, until the append method is called.
+        :param index: The index of the item in the series
```

Comment:
Why do we need `index`?
And, add an empty line between description and params - easier to read.